rules/windows/defense_evasion_clearing_windows_security_logs.toml

```diff
@@ -0,0 +1,27 @@
+[metadata]
+creation_date = "2020/11/12"
+ecs_version = ["1.6.0"]
+maturity = "development"
+updated_date = "2020/11/12"
+
+[rule]
+author = ["Anabella Cristaldi"]
+description = "Identifies attempts to clear Windows Security log stores"
+from = "now-9m"
+index = ["winlogbeat-*"]
+interval = "6m"
+language = "kuery"
+license = "Elastic License"
+max_signals = 99
```

Comment:
I think we are safe to remove this as it should be a low occurrence. The API will auto-default to a value of 100 if left undefined as well
```suggestion
```